python-lib/text_extraction/__init__.py

```diff
@@ -56,11 +56,11 @@ def extract_text_content(file_bytes, extension, with_pandoc):
         return text
 
 
-def extract_text_chunks(filename, file_bytes, extension, with_pandoc, metadata_as_plain_text):
+def extract_text_chunks(filename, file_bytes, extension, with_pandoc, metadata_as_plain_text, extract_pdf_outline):
     if extension == "pdf":
         pdf_pages = pdfium.PdfDocument(file_bytes)
         outline = list(pdf_pages.get_toc())
-        if len(outline) == 0:
+        if len(outline) == 0 or not extract_pdf_outline:
```

Comment:
Shouldn't we only consider the outline if it contains 2 items or more ?

Reply:
yes good idea, will limit to at least 2

Reply:
Actually thinking about this again, maybe it's better to let the user decide about this. For instance if there is only one bookmark but they want it as metadata instead of the page number, then that is their responsibility.

Let's revert this if you agree (and sorry for the back and forth 😅 )